### ISSUE TITLE
random detach to window while not in asbru.

### ISSUE THREAD

**Issue by NoName2Day** [asbru-cm/asbru-cm#1074]
**Describe the bug**
With asbru with multiple tabs/connections, when using another application (firefox, slack etc). asbru randomly detaches a terminal to a window. The behaviour is the same as "Detach tab to new window" function. 
This happens mostly when asbru is running maximized in the background.

**To Reproduce**
1. Open multiple sessions in asbru (more than 4 tabs)
2. Use another application, asbru randomly detaches a window, which get's opened in the foreground. It appears to always be the 3rd tab that gets disconnected. 

I am starting asbru through a hotkey with a command to op en a specific window. Closing asbru, and restarting asbru ussually fixes the issue. 

**Expected behavior**
Asbru not randomly detaching the 3rd tab while being inactive. 

**Environment (please complete the following information):**
 - OS: Kubuntu 23.04 and 23.10 both up2 date, didn't see this behaviour before 23.04 versions.
 - Ásbrú Version: currently 6.4.0

**Additional context**
It started somewhere in 23.04, this machine has been upgraded from kubuntu 22.04. 
There is a KDE bug, which is fixed in plasma 6, that you cannot start an appimage from dolphin, but from command line it works fine. This might be a simmular issue? but i don't now where to start looking for this issue.

**Comment by gfrenoy** [comment]
Well, it cannot be "random", there is a reason behind and without understanding that reason, it won't be possible to fix it...

Would you have a screencast or a more detailed description of when it happens ?

**Comment by NoName2Day** [comment]
I know, but It appears to be random, haven't been able to put a finger on it. Because while I use asbru every day. It only happens like 1 to 2 times a week. Usually when I'm working.

I've upgraded to 24.04 through a complete reinstall. I couldn't install asbru deb on 24.04, so I switched to the snap version. To my knowledge, it hasn't happend with this installation.
I will be switching back to the deb versions if there is a working version. (I don't like snap for desktop apps, they really don't integrate well and are slow).
